src/components/svgs/nodes/master.tsx

```diff
@@ -0,0 +1,298 @@
+                        ...attrs.components.map(s => s.label),
+                        'color'
+                    )(
+                        ...varValues.map((v, varI) =>
+                            varType[varI] === 'number' && !Number.isNaN(Number(v)) ? Number(v) : v
+                        ),
+                        attrs.color ? attrs.color.value ?? attrs.color.defaultValue : ''
+                    );
+                } catch (e) {
+                    modifiedAttrs[key] = '' as any;
+                }
+            }
+        }
+
+        return modifiedAttrs as T;
+    };
+
+    const gPointerEvents =
+        attrs.nodeType === 'MiscNode' ? { onPointerDown, onPointerMove, onPointerUp, style: { cursor: 'move' } } : {};
+
+    const dfsCreateElement = (svgs: MasterSvgsElem[]): ReactNode => {
+        return svgs.map(s => {
+            const coreProps =
+                attrs.nodeType === 'Station' && attrs.core && attrs.core === s.id
+                    ? { id: `stn_core_${id}`, onPointerDown, onPointerMove, onPointerUp, style: { cursor: 'move' } }
+                    : {};
+            const calcAttrs = modifyAttributes(
+                s.attrs,
+                attrs.components.map(s => s.value),
+                attrs.components.map(s => s.type)
+            );
+            return (
+                <g key={s.id} transform={`translate(${calcAttrs.x ?? 0}, ${calcAttrs.y ?? 0})`}>
+                    {React.createElement(
+                        s.type,
+                        {
+                            ...calcAttrs,
+                            x: 0,
+                            y: 0,
+                            ...coreProps,
+                        },
+                        s.children
+                            ? dfsCreateElement(s.children)
+                            : '_rmp_children_text' in calcAttrs
+                              ? (calcAttrs._rmp_children_text as string)
+                              : null
+                    )}
+                </g>
+            );
+        });
+    };
+
+    const masterTransform = attrs.transform ?? defaultMasterTransform;
+
+    return React.createElement(
+        'g',
+        { id: id, transform: `translate(${x}, ${y})`, ...gPointerEvents },
+        attrs.randomId !== 'undefined' ? (
+            <g
+                transform={`translate(${masterTransform.translateX}, ${masterTransform.translateY}) scale(${masterTransform.scale}) rotate(${masterTransform.rotate})`}
+            >
+                {dfsCreateElement(attrs.svgs)}
+            </g>
+        ) : (
+            <>
+                <circle r={5} />
+                <text x={-3.75} y={3} fill="white" fontSize="8" fontWeight="bold">
+                    M
+                </text>
+            </>
+        )
+    );
+};
+
+export interface MasterAttributes extends MasterParam {}
+
+const defaultMasterAttributes: MasterAttributes = {
+    randomId: 'undefined',
```

Comment:
This is weird. Why it is not just `undefined`?

Reply:
displaying a `undefined` in Specific Attributes ('Master type') may be not convenient (always adding `?? 'Undefined'`)

Reply:
That's not helpful when adding i18n translation. Better not hard code any user-visible text. Also why label capitalized the `Undefined` but randomId not?

Reply:
They are in a same id so user can operate them in master node manager.